python/dgl/distributed/dist_tensor.py

```diff
@@ -181,6 +181,16 @@ def __setitem__(self, idx, val):
         # TODO(zhengda) how do we want to support broadcast (e.g., G.ndata['h'][idx] = 1).
         self.kvstore.push(name=self._name, id_tensor=idx, data_tensor=val)
 
+    @property
```

Comment:
why not use existing one: https://github.com/dmlc/dgl/blob/72781efbcf2532254957090c889ec5b7b9563245/python/dgl/distributed/dist_tensor.py#L233-L242

Reply:
It's not identical to `self._name`, which is used as the key of the local KVStore dictionary.